test/cli.ava.js

```diff
@@ -0,0 +1,74 @@
+import Store from './store.js'
+import l4js from 'log4js'
+import {main} from '../lib/cli.js'
+import path from 'path'
+// eslint-disable-next-line node/no-missing-import
+import test from 'ava'
+import url from 'url'
+
+const processArgs = process.argv.slice(0, 2)
+const __filename = url.fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const examples = path.resolve(__dirname, '..', 'examples')
+const fixtures = path.resolve(__dirname, 'fixtures')
+const TEST = path.join(examples, 'test.wsd')
+
+async function run(...argv) {
+  const stdout = new Store()
+  const stderr = new Store()
+  let error = null
+
+  try {
+    await main([...processArgs, ...argv], {stdout, stderr})
+  } catch (er) {
+    error = er
+  }
+
+  return {
+    stdout: stdout.read().toString(),
+    stderr: stderr.read().toString(),
+    error,
+  }
+}
+
+test.before('Capture log events', () => {
+  l4js.configure({
+    appenders: {test: {type: 'recording'}},
+    categories: {default: {appenders: ['test'], level: 'WARN'}},
+  })
+})
+
+test.after('Reset logging', () => {
+  l4js.configure({
```

Comment:
What I want is either a `reset` method or the ability to read the current config so I can put it back.  Not having either leaves me questioning the log4js library.